assets/tools_samplesheet.csv

```diff
@@ -5,3 +5,4 @@ pcorbshrink,,,propr,--metric pcor.bshrink,,
 propr,,,propr,--metric rho,,
 cor,,,propr,--metric cor,,
 propd_grea,propd,,,,grea,
+deseq2,deseq2,,,,gsea,
```

Comment:
deseq2 does not do anything yet here no?

Reply:
No, just a placeholder for how it would look like. And with your new additions it would become functional, right?

Reply:
yes, with the new additions it would be good!